Assets/Scripts/DialogueSystem/DialogueManager.cs

```diff
@@ -127,6 +139,12 @@ void OpenMemoryMatch()
         SceneManager.LoadScene(sceneName: "MemoryMatch 1");
     }
 
+    void OpenFinalBossBattle()
+    {
+        FindObjectOfType<AudioManager>().StopPlaying("Hub_Music");
+        SceneManager.LoadScene(sceneName: "FinalBoss");
```

Comment:
since you have a few cases. I'm wondering if it would have been worthwhile to add a SceneToLoad string to the dialog or conversations object that when isn't an empty string it loads that scene when the convo is finished. Would reduce having to make individual functions/conditions.